### ISSUE TITLE
Unable to run tests

### ISSUE THREAD

**Issue by CameronEzraJones** [librariesio/libraries.io#2124]
Thanks for taking the time to raise an issue. This template should guide you through the process of submitting a bug, enhancement or feature request. Please erase any part of this template that is not relevant to your issue. 

## Bugs
Before submitting a bug report:

- [x] Double-check that the bug is persistent,
- [x] Double-check the bug hasn't already been reported [on our issue tracker](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+org%3Alibrariesio), they *should* be labelled `bug` or `bugsnag`.

If you have completed those steps then please replace this section with a description of the steps taken to recreate the bug, the expected behavior and the observed behavior.

---

This was attempted on a ~~Ubuntu 16.04~~ Debian Stretch Vagrant box that I set up. I set up the tools and dependencies as follows

- Ruby 2.5.0
- PostgreSQL 10
- ElasticSearch 2.4.6
- Redis 4.0.9

My Vagrant file can be found [here](https://gist.github.com/CameronEzraJones/cfce1df257effc3f65fc388c954764e1), and my startup script can be found [here](https://gist.github.com/CameronEzraJones/907e41261c694cc66c9e71a447f318cd)

After doing `bundle exec rake db:create db:migrate` I can run `bundle exec rails c` and `bundle exec rails s` just fine. When I try to run the tests, however, this is what I'm getting
```
$ bundle exec rake db:test:prepare
rake aborted!
ActiveRecord::StatementInvalid: PG::UndefinedObject: ERROR:  type "serial" does not exist
LINE 1: SELECT 'serial'::regtype::oid
               ^
: SELECT 'serial'::regtype::oid
/vagrant/db/schema.rb:19:in `block in <main>'
/vagrant/db/schema.rb:13:in `<main>'
/home/vagrant/.rbenv/versions/2.5.0/bin/bundle:23:in `load'
/home/vagrant/.rbenv/versions/2.5.0/bin/bundle:23:in `<main>'

Caused by:
PG::UndefinedObject: ERROR:  type "serial" does not exist
LINE 1: SELECT 'serial'::regtype::oid
               ^
/vagrant/db/schema.rb:19:in `block in <main>'
/vagrant/db/schema.rb:13:in `<main>'
/home/vagrant/.rbenv/versions/2.5.0/bin/bundle:23:in `load'
/home/vagrant/.rbenv/versions/2.5.0/bin/bundle:23:in `<main>'
Tasks: TOP => db:test:load_schema
(See full trace by running task with --trace)
```

**Comment by andrew** [comment]
Looks like this is caused by https://github.com/SchemaPlus/schema_plus_core/issues/22

An alternative way to prepare the test db is with the following command:

    bin/rails db:create db:migrate RAILS_ENV=test

Also if you drop me an email to andrew@libraries.io I'll get you added to the slack, no idea what's going on with our autoinvite system cc @katzj

**Comment by CameronEzraJones** [comment]
This worked. Thanks!

I also sent you an email regarding the slack channel.

**Comment by alencarandre** [comment]
If it is caused by SchemaPlus, I did one monkey patch to temporally solve it

https://github.com/SchemaPlus/schema_plus_core/issues/22

**Comment by katzj** [comment]
This should be fixed now that we've upgraded to rails 5.2
